cmd/evm/internal/t8ntool/execution.go

```diff
@@ -423,7 +425,6 @@ func calcDifficulty(config *params.ChainConfig, number, currentTime, parentTime
 }
 
 func writeTraceResult(tracer *tracers.Tracer, f io.WriteCloser) error {
-	defer f.Close()
```

Comment:
I don't think this is a properly way. if any error when writing then f was not closed

Reply:
A mistake change. Reverted.